components/fieldlabel/index.css

```diff
@@ -59,39 +95,49 @@ governing permissions and limitations under the License.
   font-smoothing: subpixel-antialiased;
 }
 
+/* international lang support */
+.spectrum-FieldLabel {
+  &:lang(ja),
+  &:lang(zh),
+  &:lang(ko) {
+    line-height: var(--mod-fieldlabel-CJK-line-height, var(--spectrum-fieldlabel-CJK-line-height));
+  }
+}
+
 .spectrum-FieldLabel-requiredIcon {
   margin-block: 0; /* Previously used this improperly, it doesn't reposition asterisk, just increased top margin on the whole field label var(--spectrum-fieldlabel-asterisk-margin-y) */
-  margin-inline: var(--spectrum-fieldlabel-asterisk-gap) 0;
+  margin-inline: var(--mod-fieldlabel-asterisk-gap, var(--spectrum-fieldlabel-asterisk-gap)) 0;
 }
 
 .spectrum-FieldLabel--left {
   display: inline-block;
-  padding-block: var(--spectrum-fieldlabel-m-side-padding-top) 0;
-  padding-inline: 0 var(--spectrum-fieldlabel-m-side-padding-right);
+  padding-block: var(--mod-fieldlabel-side-padding-top, var(--spectrum-fieldlabel-side-padding-top)) 0;
+  padding-inline: 0 var(--mod-fieldlabel-side-padding-right, var(--spectrum-fieldlabel-side-padding-right));
 
   & .spectrum-FieldLabel-requiredIcon {
-    margin-block: var(--spectrum-fieldlabel-m-side-asterisk-margin-y) 0;
-    margin-inline: var(--spectrum-fieldlabel-asterisk-gap) 0;
+    margin-block: var(--mod-field-label-text-to-asterisk, var(--spectrum-field-label-text-to-asterisk)) 0;
+    margin-inline: var(--mod-fieldlabel-asterisk-gap, var(--spectrum-fieldlabel-asterisk-gap)) 0; 
   }
 }
 
 .spectrum-FieldLabel--right {
   display: inline-block;
   text-align: end;
-  padding-block: var(--spectrum-fieldlabel-m-side-padding-top) 0;
-  /* todo: correct missing DNA variable */
-  padding-inline: 0 var(--spectrum-fieldlabel-m-side-padding-right);
+  padding-block: var(--mod-fieldlabel-side-padding-to, var(--spectrum-fieldlabel-side-padding-top))0; 
+  padding-inline: 0 var(--mod-fieldlabel-side-padding-right), var(--spectrum-fieldlabel-side-padding-right));
 }
 
 .spectrum-Form {
-  --spectrum-tableform-border-spacing: 0 var(--spectrum-global-dimension-size-300);
-  --spectrum-tableform-margin: calc(var(--spectrum-global-dimension-size-250) * -1) 0;
-  --spectrum-tableform-labelsabove-item-gap: var(--spectrum-global-dimension-size-100);
+  /* for /docs/form.html to set field-label inside form */
+  --spectrum-tableform-border-spacing: 0 var(--spectrum-spacing-300);
+  /* matching 20px, missing global token for 20px */
+  --spectrum-tableform-margin-calc: calc(var(--mod--spacing-300, var(--spectrum-spacing-300)) + var(--mod--spacing-300, var(--spectrum-spacing-200)));
```

Comment:
This is a place where we won't want to define the `--mod-` custom properties. Instead, we'd define them later when we actually use `--spectrum-tableform-margin-calc`. Does that make sense?

Reply:
simplified!